spec/models/work_spec.rb

```diff
@@ -26,32 +26,60 @@
     expect(work.attached_files).to be_present
   end
 
-  it 'has a valid contact email' do
-    work.contact_email = 'notavalidemail'
-    expect { work.save! }.to raise_error(ActiveRecord::RecordInvalid, /Contact email is invalid/)
-  end
+  describe 'contact_email' do
```

Comment:
Does this belong in this PR?

Reply:
It was changed to be consistent with how it was being done in this PR.